src/goto-symex/symex_assign.h

```diff
@@ -35,7 +36,9 @@ class symex_assignt
       assignment_type(assignment_type),
       ns(ns),
       symex_config(symex_config),
-      target(target)
+      target(target),
+      char_type(16),
+      length_type(32)
```

Comment:
these fields shouldn't be necessary and this is not language independent like symex should be.